mailbox/store/src/main/java/org/apache/james/mailbox/store/search/SearchSnippet.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.james.mailbox.store.search;
+
+public record SearchSnippet(String id,
```

Comment:
`MessageId` or `MessageUid` is better here?
MessageUid?

Reply:
MessageId